### ISSUE TITLE
Default settings make the lock icon invisible

### ISSUE THREAD

**Issue by brisingre** [kamyker/UnityDarkenThemer#5]
![image](https://user-images.githubusercontent.com/7477052/88543004-2f2d5700-cfcc-11ea-87c7-cc669e74549c.png)
Other than this one issue this is a very nice theme, but it's so hard to see this icon...

**Comment by kamyker** [comment]
True, no idea how to fix it, Unity prob hardcoded that one :/. Maybe in new versions (2020) it will work as they are upgrading to UIElements slowly.

**Comment by brisingre** [comment]
Hopefully...

I had hope for a minute that I might be able to make it visible by tweaking the color of the bar behind it.

But Unity chose a color that really doesn't stand out well on any background, I have not had much success.
